AIPscan/Reporter/templates/reports.html

```diff
@@ -63,143 +63,136 @@
           </tr>
       {% endif %}
 
+      <tr>
+          <td>
+              <strong>AIPs by PUID</strong>
+              <br>
+              <select id="originalPUIDSelect" method="GET" action="/" class="form-control" style="width:90%;">
+              {% for puid in original_puids %}
+                <option value="{{ puid }}">{{ puid }}</option>
+              {% endfor %}
+            </select>
+          </td>
+          <td>
+            <span class="text-muted">AIPs containing original files matching the selected PRONOM Unique Identifier (file format versions)
+              <br>Filter by storage location.</span>
+          </td>
+          <td><a href="#"><button type="button" id="aipsByOriginalPUID" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+      </tr>
+
       {% if preservation_puids %}
           <tr>
               <td>
-                  <strong>AIPs by preservation format version</strong>
+                  <strong>AIPs by preservation copy PUID</strong>
                   <br>
-                  <span class="text-muted">List all AIPs that contain the selected preservation format version (PUID), filter by Storage Location</span>
-              </td>
-              <td>
-                <select id="preservationPUIDSelect" method="GET" action="/" class="form-control" style="width:90%;">
+                  <select id="preservationPUIDSelect" method="GET" action="/" class="form-control" style="width:90%;">
                   {% for puid in preservation_puids %}
                     <option value="{{ puid }}">{{ puid }}</option>
                   {% endfor %}
                 </select>
               </td>
-              <td><a href="#"><button type="button" id="aipsByPreservationPUID" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+
+          <td>
+            <span class="text-muted">AIPs containing preservation copies matching the selected PRONOM Unique Identifier (file format versions)
+              <br>Filter by storage location.</span>
+          </td>
+
+          <td><a href="#"><button type="button" id="aipsByPreservationPUID" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
           </tr>
       {% endif %}
 
       <tr>
-          <td><strong>Preservation derivatives</strong>
-            <br>
-            <span class="text-muted">Mapping original files to their preservation copy counterpart</td>
-          <td><span class="text-muted">Filter by storage location
-            <br>Filter by AIP</td>
+          <td><strong>Preservation derivatives</strong></td>
+          <td><span class="text-muted"><span class="text-muted">Map original files to their preservation copy counterpart.
+            <br>Filter by storage location and filter by AIP (inside report)</td>
```

Comment:
This is a nice way of handling this. Maybe the in-report filter notes could be on their own line? to make the distinction a bit clearer?